platform/linux-generic/odp_packet_io.c

```diff
@@ -50,6 +50,9 @@
 /* Max wait time supported to avoid potential overflow */
 #define MAX_WAIT_TIME (UINT64_MAX / 1024)
 
+/* One hour maximum aging timeout */
+#define MAX_TX_AGING_TMO_NS 3600000000000ULL
```

Comment:
If there is some reason for limiting the timeout to this value, it could be mentioned here.

Reply:
No reason really, just some value -> opened up the fudge factorness with a comment.